app/views/admin/truework_income_validations/_truework_income_validation.html.haml

```diff
@@ -0,0 +1 @@
+= render partial: 'admin/validations/validation', locals: {validation: truework_income_validation}

```

Comment:
I think this file needs to be called app/views/admin/true_work_income_validations/_true_work_income_validation.html.haml (i.e. `true_work` instead of `truework`)?  With the current name, I see errors about missing files.

Reply:
I'm also a little confused about naming.  There was a pre-existing folder called `true_work_income_validations`, and you added a new one called `truework_income_verifications`.  The actual classname is currently `TrueWorkIncomeValidation` (which we can change, but we should probably change it in a consistent manner?)